lib/tst_safe_macros.c

```diff
@@ -625,3 +625,17 @@ void safe_print_file(const char *file, const int lineno, char *path)
 
 	safe_fclose(file, lineno, NULL, pfile);
 }
+
+int safe_sscanf(const char *file, const int lineno, const char *restrict buffer, const char *restrict format, ...)
+{
+	va_list args;
+	va_start(args, format);
+	int ret = vsscanf(buffer, format, args);
+	va_end(args);
+	int placeholders=tst_count_scanf_conversions(format);
+	if (ret != placeholders)
```

Comment:
nit: blank line before `if ()` and `return` also helps readability (also makes sources a bit consistent - it's used in whole file).